openapi/doc-ru-config-values.yaml

```diff
@@ -88,6 +88,12 @@ properties:
                   bucket:
                     description: |
                       Контейнер, в котором вы можете хранить свои файлы и объекты данных.
+  virtualImages:
+    type: object
+    required: [storageClassName]
+    properties:
+      storageClassName:
+        type: string
```

Comment:
Translate file should not contains "required" and types,  it should contains translated "description" fields and "properties"/"items".

Reply:
Fixed.